src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private static final Color[] ALL_COLORS = Color.values();
+    private static final Random RANDOM = new Random();
```

Comment:
static final doesn't automatically mean constant, object has to be deeply immutable. Let's follow dependency injection pattern and remove static here